src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -738,10 +740,12 @@ internal override byte[] GetLocalsSignature()
             return m_localSignature;
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
The base class annotation should be fixed instead.

Reply:
FIxed.